src/components/user-profile.component.js

```diff
@@ -32,6 +33,9 @@ export const UserProfile = ({
       />
       <Text style={styles.title}>{user.name}</Text>
       <Text style={styles.subtitle}>{initialUser.login}</Text>
+      <Text style={styles.bio}>
+        {get(initialUser, ['bio', 'length'], 0) ? initialUser.bio : user.bio}
```

Comment:
I like `lodash.get`. We can consider adding the entire library if we need more utilities but for now `.get` is definitely useful :)